src/libServer/Server.cpp

```diff
@@ -92,26 +91,16 @@ Json::Value Server::getLatestTxBlock()
 
 Json::Value Server::getBalance(const string & address)
 {
-	return "Hello";
-	// vector<unsigned char> input_address = DataConversion::HexStrToUint8Vec(address);
-	// PrivKey privKey(tmpprivkey, 0);
-
-	// std::vector<unsigned char> input_address(address.begin(), address.end());
-	// const PrivKey priv(input_address, 0);
-	// const PubKey pub(priv);
-	// Address addr = Account::GetAddressFromPublicKey(pub);
-	// Account* account = AccountStore::GetInstance().GetAccount(addr);
-
-	// if(account != nullptr)
- //    {
- //    	Json::Value ret;
- //        ret["balance"] = 24;//account->GetBalance();
- //        ret["nonce"] = 0;//account->GetNonce();
+	vector<unsigned char> input_address(address.begin(), address.end());
+	const PrivKey priv(input_address, 0);
```

Comment:
Don't use magic number. Consider the following 
unsigned int offset = 0; 
const PrivKey priv(input_address, offset);